okhttp/src/main/java/okhttp3/internal/platform/Platform.kt

```diff
@@ -188,27 +189,27 @@ open class Platform {
   override fun toString(): String = javaClass.simpleName
 
   companion object {
-    private val PLATFORM = findPlatform()
+    private var platform = findPlatform()
 
     const val INFO = 4
     const val WARN = 5
 
     private val logger = Logger.getLogger(OkHttpClient::class.java.name)
 
     @JvmStatic
-    fun get(): Platform = PLATFORM
+    fun get(): Platform = platform
+
+    @TestOnly
```

Comment:
FWIW, I’d prefer the method name be like `resetForTests` so the test-only nature of it is visible at the callsite. I find `@TestOnly` and `@VisibleForTesting` to be weird because they exist to discourage future changes that might otherwise make sense.